src/entities/Client.ts

```diff
@@ -0,0 +1,38 @@
+import { Entity, Column, BeforeInsert, BeforeUpdate, CreateDateColumn, UpdateDateColumn, JoinColumn, ManyToOne, PrimaryColumn, BaseEntity} from "typeorm";
+import * as bcrypt from "bcrypt";
+import { Role } from "./Role";
+@Entity('users')
+export class Client extends BaseEntity {
+    @PrimaryColumn({
+        unique: true
+    })
+    user_id!: number;
+    
+    @Column({ unique: true })
+    username!: string;
+
+    @Column()
+    name!: string;
+
+    @Column()
+    password!: string;
+
+    @BeforeInsert()
+    @BeforeUpdate()
+    async hashPassword() {
+        if (this.password) {
+            this.password = await bcrypt.hash(this.password, 10);
+        }
+    }
```

Comment:
you might consider adding an email to the user entity

Reply:
Perfect, will add it